<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqPipelineBrowserContextMenu</class>
 <widget class="QWidget" name="pqPipelineBrowserContextMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>30</height>
   </rect>
  </property>
  <action name="actionPBChangeInput">
   <property name="text">
    <string>Изменить &amp;входные данные...</string>
   </property>
   <property name="iconText">
    <string>Изменить входные данные...</string>
   </property>
   <property name="toolTip">
    <string>Изменить входные данные фильтра</string>
   </property>
   <property name="statusTip">
    <string>Изменить входные данные фильтра</string>
   </property>
  </action>
  <action name="actionPBDelete">
   <property name="icon">
    <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
     <normaloff>:/QtWidgets/Icons/pqDelete24.png</normaloff>:/QtWidgets/Icons/pqDelete24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Удалить</string>
   </property>
   <property name="statusTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionPBCopy">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqCopy22.png</normaloff>:/pqWidgets/Icons/pqCopy22.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Копировать</string>
   </property>
   <property name="statusTip">
    <string>Копировать свойства</string>
   </property>
  </action>
  <action name="actionPBPaste">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqPaste22.png</normaloff>:/pqWidgets/Icons/pqPaste22.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Вставить</string>
   </property>
   <property name="statusTip">
    <string>Вставить свойства</string>
   </property>
  </action>
  <action name="actionPBIgnoreTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Не учитывать время</string>
   </property>
   <property name="toolTip">
    <string>Не учитывать время данного объекта/фильтра в анимации</string>
   </property>
   <property name="statusTip">
    <string>Не учитывать время данного объекта/фильтра в анимации</string>
   </property>
   <property name="whatsThis">
    <string>Не учитывать время данного объекта/фильтра в анимации</string>
   </property>
  </action>
  <action name="actionPBOpen">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqOpen24.png</normaloff>:/pqWidgets/Icons/pqOpen24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть</string>
   </property>
   <property name="statusTip">
    <string>Открыть</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionPBCreateCustomFilter">
   <property name="text">
    <string>&amp;Создать пользовательский фильтр...</string>
   </property>
  </action>
  <addaction name="actionPBOpen"/>
  <addaction name="actionPBCopy"/>
  <addaction name="actionPBPaste"/>
  <addaction name="actionPBCreateCustomFilter"/>
  <addaction name="actionPBChangeInput"/>
  <addaction name="actionPBIgnoreTime"/>
  <addaction name="actionPBDelete"/>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
